platform/linuxbsd/x11/display_server_x11.h

```diff
@@ -156,6 +156,7 @@ class DisplayServerX11 : public DisplayServer {
 #ifdef SPEECHD_ENABLED
 	TTS_Linux *tts = nullptr;
 #endif
+	NativeMenu *native_menu = nullptr;
```

Comment:
Some dummy instance should exist, since it can be accessed in any platform. Also, even if we won't have global menu support we might implement native popup menu at some point (can be useful for the tray indicator menus, and I think the only way tray indicators can be done on X11).